Backend/src/main/java/com/pg/programmerground/controller/PlaygroundController.java

```diff
@@ -22,38 +26,55 @@
      * 무한 스크롤 적용위한 방법? 찾기
      */
     @GetMapping("")
-    public ResponseEntity<ApiResponse<List<PlaygroundCardInfoDto>>> playgroundList() {
+    public ResponseEntity<ApiResponse<List<PlaygroundCardResponse>>> playgroundList() {
```

Comment:
```suggestion
    public ResponseEntity<ApiResponse<PlaygroundCardListResponse>> playgroundList() {
```

위에처럼 리턴을 하면

```json
{
  "service":"programmers",
  "code":200,
  "data":[{},{}]
}
```
이렇게 될 것 같은데, 가령 응답 데이터에 필드가 추가된다면 data 프로퍼티 구조 전체가 변경될 가능성이 있음 (ex: 무한 스크롤을 위해, 페이징 정보가 응답 데이터에 추가되는경우).  
data 프로퍼티는 오브젝트로 리턴하는게 더 안전함.
```json
{
  "service":"programmers",
  "code":200,
  "data":{
    "playgrounds":[{},{}]
  }
}
```